cmake/kokkos_arch.cmake

```diff
@@ -838,22 +838,49 @@ ENDIF()
 
 IF (KOKKOS_ENABLE_OPENACC)
   IF(KOKKOS_CUDA_ARCH_FLAG)
+    IF(KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE)
+      MESSAGE(WARNING "If a GPU architecture is specified, Kokkos_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE option will be ignored.")
```

Comment:
I think that should be a hard error (same comment in the AMDGPU branch)

Reply:
Changed.